api/assemblyline/assemblyline.py

```diff
@@ -0,0 +1,153 @@
+from assemblyline_client import get_client
+from boto3wrapper.wrapper import get_session
+from database.db import get_db_session
+from database.dynamodb import log_scan_result
+from json import dumps
+from logger import log
+from models.Scan import Scan, ScanProviders, ScanVerdicts
+from os import environ
+from storage.storage import get_file
+
+
+def get_assemblyline_client():
+    return get_client(
+        f"https://{environ.get('MLWR_HOST')}",
+        timeout=60000,  # Timeout after 1 minute
+        apikey=(environ.get("MLWR_USER"), environ.get("MLWR_KEY")),
+    )
+
+
+def add_to_scan_queue(payload):
+    client = get_session().client("stepfunctions")
+    response = client.list_state_machines()
+    state_machine = "assemblyline-file-scan-queue"
```

Comment:
minor nit: extract out into file level constant like `dynamodb.DYNAMODB_TABLE `

Reply:
moved to environment variable and TF level variable so that i can have it declared in one place.